tests/services/test_ses.py

```diff
@@ -0,0 +1,94 @@
+from unittest.mock import (
+    AsyncMock,
+)
+
+import pytest
+from aiobotocore.client import AioBaseClient
+
+from auth_app.config import aws_settings
+from auth_app.schemes.email import EmailPayloadScheme
+from auth_app.services.ses.ses_handler import ses_handler
+
+
+@pytest.fixture
+def generate_email_payload_data() -> dict:
+    return {
+        "message": "Test message",
+        "subject": "Test subject",
+    }
+
+
+@pytest.fixture
+def send_email_patch(
+    mocker,
+) -> AsyncMock:
+    send_email = mocker.patch(
+        "auth_app.services.ses.ses_handler.ses_handler.send_email",
+        new_callable=AsyncMock,
+        return_value=None,
+    )
+    return send_email
+
+
+def test_generate_email_payload(
+    generate_email_payload_data,
+) -> None:
+    """
+    Test of valid email payload creation
+    """
+    email_payload = ses_handler.generate_email_payload(
+        **generate_email_payload_data,
+    )
+
+    assert isinstance(email_payload, EmailPayloadScheme)
+    assert email_payload.message == "Test message"
+    assert email_payload.subject == "Test subject"
```

Comment:
непонятно откуда данные вылазят
правильнее было бы так 
```suggestion
    assert email_payload.message == generate_email_payload_data["message"]
    assert email_payload.subject == generate_email_payload_data["subject"]
```

Reply:
полностью поддерживаю, проглядел